packages/manager/core/generator/app/conditional-templates/default/home.test.ts.hbs

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from '@playwright/test';
+import { login, logout } from '../../../../../playwright/login';
+import * as config from '../../../../../playwright/config.json';
+
+test('Listing page test', async ({ page }) => {
+  await page.goto(config.appUrl);
+  await login(page);
+  await page.waitForSelector('body');
+  const pageContent = await page.textContent('body');
+  await expect(pageContent).toContain('Start your application');
```

Comment:
you should fuse theses 3 lines in one something like await expect(page.locator('Start you application').toBeVisible();

Reply:
Done ✅